core/src/main/java/hudson/model/StreamBuildListener.java

```diff
@@ -52,12 +55,6 @@ public StreamBuildListener(OutputStream w) {
         super(w);
     }
 
-    /**
-     * @deprecated as of 1.349
-     *      The caller should use {@link #StreamBuildListener(OutputStream, Charset)} to pass in
-     *      the charset and output stream separately, so that this class can handle encoding correctly.
-     */
-    @Deprecated
```

Comment:
I do not think there is any reason to touch this class, which could be used in other places. Just fix the call site in `Run` to specify `Charsets.UTF-8`.